src/legacy/core_plugins/data/public/search/search_bar/lib/saved_query_service.test.ts

```diff
@@ -140,6 +140,15 @@ describe('saved query service', () => {
       }
       expect(error).not.toBe(null);
     });
+    it("should throw an error if the saved query doesn't have a title", async () => {
+      let error = null;
+      try {
+        await saveQuery({ ...savedQueryAttributes, title: '' });
+      } catch (e) {
+        error = e;
+      }
+      expect(error).not.toBe(null);
```

Comment:
FYI Jest has a helper to make testing errors easier https://jestjs.io/docs/en/expect#tothrowerror